router/core/request_context_fields.go

```diff
@@ -69,17 +69,20 @@ func NewDurationLogField(val time.Duration, attribute config.CustomAttribute) za
 }
 
 func AccessLogsFieldHandler(attributes []config.CustomAttribute, err any, request *http.Request, responseHeader *http.Header) []zapcore.Field {
-	reqContext := getRequestContext(request.Context())
-
 	resFields := make([]zapcore.Field, 0, len(attributes))
-	resFields = append(resFields, logging.WithRequestID(middleware.GetReqID(request.Context())))
+
+	var reqContext *requestContext
```

Comment:
Please add a test that validates the subgraph logs when the subgraph call errors, timeout etc...